app/views/devise/shared/_links.erb

```diff
@@ -20,6 +20,10 @@
 
 <%- if devise_mapping.omniauthable? %>
   <%- resource_class.omniauth_providers.each do |provider| %>
-    <%= link_to "Войти с помощью #{provider.to_s.titleize}", omniauth_authorize_path(resource_name, provider) %><br />
+    <%- if provider.to_s.titleize != 'Google Oauth2' %>
+      <%= link_to "Войти с помощью #{provider.to_s.titleize}", omniauth_authorize_path(resource_name, provider) %><br />
+    <%- else %>
+      <%= link_to "Войти с помощью Google", omniauth_authorize_path(resource_name, provider) %><br />
```

Comment:
It is necessary to display information for user.
Google instead of Google Oauth2.